docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -215,14 +215,35 @@ This option is only available when you're editing the panel.
 
 ### Table footer options
 
-Toggle the **Show table footer** switch on and off to control the display of the footer.
-When the toggle is switched on, you can use the table footer to show [calculations](ref:calculations) on fields.
+The table footer displays the results of calculations (and reducer functions) on fields.
+The footer is only displayed after you select an option in the **Calculation** drop-down list:
 
-After you activate the table footer, make selections for the following options:
+{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-table-footer-selector-v12.2.png" max-width="300px" alt="" >}}
 
-- **Calculation** - The calculation that you want to apply.
-- **Count rows** - This option is displayed if you select the **Count** calculation. If you want to show the number of rows in the dataset instead of the number of values in the selected fields, toggle on the **Count rows** switch.
-- **Fields** - The fields to which you want to apply the calculation. Grafana applies the calculation to all numeric fields if you don't select a field.
+There are several calculations you can choose from including minimum, maximum, first, last, and total.
+For the full list of options, refer to [Calculations](ref:calculations).
+
+In the table footer:
+
+- You can apply multiple calculations at once.
+- All calculations and reducer functions are labeled except **Total**.
+- The calculations and reducer functions apply to all fields in the table, by default. To control which fields have a calculation or function applied, add the table footer in an override instead.
+- If you enable a mathematical function for a non-numeric field, nothing for that function is displayed for that field.
+
+In the following image, multiple calculations&mdash;**Mean**, **Max**, and **Last**&mdash;have been applied:
+
+{{< figure src="/media/docs/grafana/panels-visualizations/screenshot-table-footer-1-v12.2.png" max-width="750px" alt="" >}}
+
+You can also see in the previous image that the mathematical functions, **Mean** and **Max**, haven't been applied to the text field in the table.
+Only the **Last** function has been applied to that field.
+
+In the following image, the **Total** calculation has been applied, and no label is displayed:
```

Comment:
```suggestion
In the following image, the **Total** calculation has been applied, and no label is displayed because it's the only calculation applied:
```